applications/nrf5340_audio/src/bluetooth/bt_content_control/content/bt_media_control.c

```diff
@@ -184,101 +236,172 @@ static void mcs_local_player_instance_cb(struct media_player *player, int err)
 	(void)media_proxy_ctrl_send_command(local_player, &cmd);
 }
 
-int ble_mcs_discover(struct bt_conn *conn)
+/**
+ * @brief	Send play or pause command to either local media player or peer
+ *
+ * param	cmd	Command to send
+ *
+ * @return	0 for success, error otherwise.
+ */
+static int play_pause(struct mpl_cmd *cmd)
+{
+	int ret = 0;
+
+	if (IS_ENABLED(CONFIG_BT_MCS)) {
```

Comment:
Would it make sense to verify that the `cmd.opcode` is indeed a play or pause?

Reply:
Should probably rather rename the function to mpl_cmd_send instead since this function just sends the cmd it receives to the proper destination, being either the local media player or over the air